### ISSUE TITLE
Time format 24H

### ISSUE THREAD

**Issue by Shto1337** [cronicle-edge/cronicle-edge#140]
Hello, 

Inspired from https://github.com/jhuckaby/Cronicle/issues/250 , Is it possible to display time values in 24 hour format instead of AM/PM?

**Comment by mikeTWC1984** [comment]
I guess that kinda doable. As an easy hack, you can just override  get_nice_time function, e.g. by adding new definition on index.html (index-bundle.html) after loading common.min.js  It will not change it all over the place, but in many cases it will work.

```html
	<script>	
		function get_nice_time(epoch, secs) {
			// return time in HH12:MM format
			var dargs = get_date_args(epoch);
			if (dargs.min < 10) dargs.min = '0' + dargs.min;
			if (dargs.sec < 10) dargs.sec = '0' + dargs.sec;
			var output = dargs.hour + ':' + dargs.min;
			if (secs) output += ':' + dargs.sec;
			// output += ' ' + dargs.ampm.toUpperCase();
			return output;
		}
	</script>
``` 

I might add this option later to this repo

**Comment by mikeTWC1984** [comment]
Implemented this in 0.10.0 version (just released). You can set ui.hh24 config to true (check release notes for more info)
